docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5381,4 +5381,21 @@ entity {
 ~ End Prototext
 
 
+## Guidelines for Implementations
+
+This section contains practical advice for using P4Runtime.
+
+### gRPC Metadata Maximum Size
+
+In gRPC, the status of a RPC request is sent as metadata, whose size is limited by the `grpc.max_metadata_size` gRPC channel argument.  By default, this limit is 8KB, which can be a problem for the `Write` P4Runtime RPC.  The `Write` RPC returns an individual error for every item in a batch (see Section [#sec-write-rpc]), which can quickly result in a status size over 8KB.  In that case, the gRPC server would not send the status, but instead send a `RESOURCE_EXHAUSTED` error, without any of the individual errors.
```

Comment:
Could you wrap lines to 80 characters?

Reply:
Yes, of course.  I meant to, but forgot.